src/commands/txCommands/util/pushTxAllTranslations.js

```diff
@@ -0,0 +1,12 @@
+const chalk = require('chalk');
+const Rx = require('rxjs');
+const txlib = require('./index');
+
+const pushTxAllTranslations = () => {
+  txlib.checkEnvVars();
+  console.log(chalk.blue('pushing content to all_translations in mup-web'));
+  Rx.Observable
+    .concat(txlib.updateAllTranslationsResource$)
```

Comment:
i think `Rx.Observable.concat` is only useful when you have more than one observable you want to combine